### PR TITLE
Hms merge

### PR DESCRIPTION
**Pull request by StefanBabukov**
# Description

# Details
#### URL to issue
N/A

#### Link to staging deployment URL (or set N/A)
https://ui-hms-merge-8.scp-staging.biomage.net/

#### Integration test branch
master

# Merge checklist
Your changes will be ready for merging after all of the steps below have been completed.

### Code updates
Have best practices and ongoing refactors being observed in this PR
- [ ] Migrated any selector / reducer used to the new format.

### Manual/unit testing
- [ ] Tested changes using InfraMock locally **or** no tests required for change, e.g. Kubernetes chart updates.
- [ ] Validated that current unit tests for code work as expected and are sufficient for code coverage **or** no unit tests required for change, e.g. documentation update.
- [ ] Unit tests written **or** no unit tests required for change, e.g. documentation update.

### Integration testing
**You must check the box below** to run integration tests on the latest commit on your PR branch.
Integration tests have to pass before the PR can be merged. Without checking the box, your PR
**will not pass** the required status checks for merging.

- [x] Started end-to-end tests on the latest commit.

### Documentation updates
- [ ] Relevant Github READMEs updated **or** no GitHub README updates required.
- [ ] Relevant Wiki pages created/updated **or** no Wiki updates required.

### Optional
- [ ] Staging environment is unstaged before merging.
- [ ] Photo of a cute animal attached to this PR.

### CODE REVIEW

File: .flux.yaml [hunk 1]
```diff
@@ -12,7 +12,7 @@ metadata:
   namespace: FILLED_IN_BY_CI
 spec:
   interval: 5m0s
-  url: https://github.com/biomage-org/worker
+  url: https://github.com/hms-dbmi-cellenics/worker
```
Comment (StefanBabukov) [inline .flux.yaml:15]: This

File: .github/workflows/pr_validate.yaml [hunk 2]
```diff
@@ -28,8 +28,7 @@ jobs:
       - id: check-issue-url
         name: Check issue URL
         run: |-
-          REGEX="#### URL to issue\s?\r\n(https:\/\/biomage\.atlassian\.net\/browse\/BIOMAGE-[0-9]+|https:\/\/github\.com\/biomage-org\/issues\/issues\/[0-9]+|N\/A)"
-
+          REGEX="#### URL to issue\s?\r\n(https:\/\/github\.com\/hms-dbmi-cellenics\/issues\/issues\/[0-9]+|N\/A)"
```
Comment (StefanBabukov) [inline .github/workflows/pr_validate.yaml:31]: 3 places in this file

File: .github/workflows/ci.yaml [hunk 3]
```diff
@@ -115,7 +115,7 @@ jobs:
           # Images for production uses semantic versioning to determine the latest image.
           echo "timestamp=$TIMESTAMP" >> $GITHUB_OUTPUT
 
-          # This will take a GitHub repo name like `biomage-org/iac`
```
Comment (StefanBabukov) [inline .github/workflows/ci.yaml:118]: 4 places in this  file

File: .github/pull_request_template.md
Comment (StefanBabukov) [inline .github/pull_request_template.md:1]: comments in this file can be changed too

File: README.md [hunk 4]
```diff
@@ -1,4 +1,4 @@
-[![codecov](https://codecov.io/gh/biomage-org/worker/branch/master/graph/badge.svg?token=3PHqr61GpH)](https://codecov.io/gh/biomage-org/worker)
+[![codecov](https://codecov.io/gh/hms-dbmi-cellenics/worker/branch/master/graph/badge.svg?token=3PHqr61GpH)](https://codecov.io/gh/hms-dbmi-cellenics/worker)
```
Comment (StefanBabukov) [inline README.md:1]: 3 places in this file

File: python/README.md [hunk 5]
```diff
@@ -21,8 +21,8 @@ See the main README for instructions on how to run the workers in Docker.
 ### Process tasks
 Tasks are automatically processed when they are received from the SQS queue specified.
 
-For **local development**, make sure you have [InfraMock](https://github.com/biomage-org/inframock)
```
Comment (StefanBabukov) [inline python/README.md:24]: 2 places in this file, even though they are README

File: r/setup/check_package_licenses.R [hunk 6]
```diff
@@ -0,0 +1,40 @@
+get_licenses <- function(package_dir = '.') {
```
Comment (StefanBabukov) [inline r/setup/check_package_licenses.R:1]: Do we need this too?

**Review by kafkasl — APPROVED** [review]

**Pull request merged**